provider/virtual_network/vdpa_utils.py

```
@@ -0,0 +1,132 @@
+
+    :param pf_pci: PF pci address
+    :type pf_pci: str
+
+    :return: Port name
+    :rtype: str
+    """
+    cmd = "ls /sys/bus/pci/devices/{0}/net/ | head -n 1".format(pf_pci)
+    return process.system_output(cmd, shell=True).decode().strip()
+
+
+def add_flows_to_ovs_bridge(bridge, ovs):
```

Ruff (0.14.10)

Unused function argument: `ovs`

(ARG001)